awx/ui_next/src/screens/Job/JobOutput/JobOutput.jsx

```diff
@@ -447,174 +468,300 @@ class JobOutput extends Component {
+  };
+
+  const pushHistoryState = params => {
+    const { pathname } = history.location;
+    const encodedParams = encodeNonDefaultQueryString(QS_CONFIG, params);
+    history.push(encodedParams ? `${pathname}?${encodedParams}` : pathname);
+  };
+
+  const renderSearchComponent = i18n => (
+    <Search
+      qsConfig={QS_CONFIG}
+      columns={[
+        {
+          name: i18n._(t`Stdout`),
+          key: 'stdout__icontains',
+          isDefault: true,
+        },
+        {
+          name: i18n._(t`Event`),
+          key: 'event',
+          options: [
+            ['runner_on_failed', i18n._(t`Host Failed`)],
+            ['runner_on_start', i18n._(t`Host Started`)],
+            ['runner_on_ok', i18n._(t`Host OK`)],
+            ['runner_on_error', i18n._(t`Host Failure`)],
+            ['runner_on_skipped', i18n._(t`Host Skipped`)],
+            ['runner_on_unreachable', i18n._(t`Host Unreachable`)],
+            ['runner_on_no_hosts', i18n._(t`No Hosts Remaining`)],
+            ['runner_on_async_poll', i18n._(t`Host Polling`)],
+            ['runner_on_async_ok', i18n._(t`Host Async OK`)],
+            ['runner_on_async_failed', i18n._(t`Host Async Failure`)],
+            ['runner_item_on_ok', i18n._(t`Item OK`)],
+            ['runner_item_on_failed', i18n._(t`Item Failed`)],
+            ['runner_item_on_skipped', i18n._(t`Item Skipped`)],
+            ['runner_retry', i18n._(t`Host Retry`)],
+            ['runner_on_file_diff', i18n._(t`File Difference`)],
+            ['playbook_on_start', i18n._(t`Playbook Started`)],
+            ['playbook_on_notify', i18n._(t`Running Handlers`)],
+            ['playbook_on_include', i18n._(t`Including File`)],
+            ['playbook_on_no_hosts_matched', i18n._(t`No Hosts Matched`)],
+            ['playbook_on_no_hosts_remaining', i18n._(t`No Hosts Remaining`)],
+            ['playbook_on_task_start', i18n._(t`Task Started`)],
+            ['playbook_on_vars_prompt', i18n._(t`Variables Prompted`)],
+            ['playbook_on_setup', i18n._(t`Gathering Facts`)],
+            ['playbook_on_play_start', i18n._(t`Play Started`)],
+            ['playbook_on_stats', i18n._(t`Playbook Complete`)],
+            ['debug', i18n._(t`Debug`)],
+            ['verbose', i18n._(t`Verbose`)],
+            ['deprecated', i18n._(t`Deprecated`)],
+            ['warning', i18n._(t`Warning`)],
+            ['system_warning', i18n._(t`System Warning`)],
+            ['error', i18n._(t`Error`)],
+          ],
+        },
+        { name: i18n._(t`Advanced`), key: 'advanced' },
+      ]}
+      searchableKeys={eventSearchableKeys}
+      relatedSearchableKeys={eventRelatedSearchableKeys}
+      onSearch={handleSearch}
+      onReplaceSearch={handleReplaceSearch}
+      onShowAdvancedSearch={() => {}}
+      onRemove={handleRemoveSearchTerm}
+      isDisabled={isJobRunning(job.status)}
+    />
+  );
+
+  if (hasContentLoading) {
+    return <ContentLoading />;
+  }
```

Comment:
To reduce "flashing" during search, we could move `<ContentLoading/>` further down into the Job Output. 

```
<AutoSizer nonce={window.NONCE_ID} onResize={handleResize}>
  {({ width, height }) => {
    return (
      <>
      {hasContentLoading ? (
        <div style={{ width }}>
          <ContentLoading />
        </div>
      ) : (
        <List
          ref={ref => {
            registerChild(ref);
            listRef.current = ref;
          }}
 ...
```
![output](https://user-images.githubusercontent.com/15881645/107054842-9eb3a880-679e-11eb-8762-08b0948c9e85.gif)